CinemaTicketSalesSystem/CinemaTicketSalesSystem/Views/Shared/MasterView.cshtml

```diff
@@ -7,8 +7,11 @@
 
 <html>
 <head>
-    <meta name="viewport" content="width=device-width" />
+    @Scripts.Render("~/bundles/jquery")
+    @Scripts.Render("~/bundles/chosen")
```

Comment:
only style bundles and required js files should be in head ( like jquery)
all additional 3rd party scripts you should put in the end of body